<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connect_window</class>
 <widget class="QDialog" name="connect_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connect to...</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Host IP :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Port :</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Name :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="line_host_ip"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="line_port"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="line_name"/>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="button_connect">
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="button_cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_cancel</sender>
   <signal>clicked()</signal>
   <receiver>connect_window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
